test/setup

```diff
@@ -70,8 +70,9 @@ popd
 rm -rf ${CERT_DIR}
 
 echo "booting ${VMS} by heat, then copy files for pre-deployment"
-extra_args="--ursula-debug --provisioner=heat --heat-stack-name=${testenv_heat_stack_name}"
-ursula ${extra_args} "envs/test" ${ROOT}/playbooks/${TEST_ENV}/tasks/pre-deploy.yml -u ${LOGIN_USER} "$@"
+sleep 30
```

Comment:
Booting centos images seems to take just long enough to cause a race condition where a node is not ready by the time we call ursula.